step_definitions/management_classes/deleteClass_steps.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const manageClass = require('../../page-objects/pages/managementPages/ManageClassPage');
+
+Then(/^.* clicks delete class with name '([^']*)'$/, function (ClassName) {
+	return manageClass.deleteClass(ClassName)
```

Comment:
Please be aware of our naming conventions https://docs.hpi-schul-cloud.org/display/CQA/04+JavaScript+naming+convention